conda_smithy/templates/appveyor.yml.tmpl

```diff
@@ -9,11 +9,6 @@ environment:
   # See: http://stackoverflow.com/a/13751649/163740
   CMD_IN_ENV: "cmd /E:ON /V:ON /C obvci_appveyor_python_build_env.cmd"
 
-  # We set a default Python version for the miniconda that is to be installed. This can be
-  # overridden in the matrix definition where appropriate.
-  CONDA_PY: "27"
```

Comment:
So I guess this is still in [use]( https://github.com/pelson/Obvious-CI/blob/v0.6.1/scripts/obvci_appveyor_python_build_env.cmd#L1-L2 ) by `obvci_appveyor_python_build_env.cmd`. However, it has repeatedly come up that people think this script is not needed either. Maybe it is time to remove it too.